electrumsv/main.py

```diff
@@ -277,6 +277,9 @@ def load_app_module(module_name, config):
 def run_app_with_daemon(fd, is_gui, config_options):
     with app_state.async_ as async_:
         d = daemon.Daemon(fd, is_gui)
+        if not is_gui:
```

Comment:
Why not just call this in all cases? The dummy method can be called for non-dapps.